### PR TITLE
Dev -> Prod

### PR DESCRIPTION
**Pull request by jkonrath-postman**
Update One Trust ID for Learning Center #3958 
Fix redirect vs page error #3956 
TW-64 Updated information about public collaboration #3919 
Fix redirects.json #3954 
TW-420 Update icon style #3949 
TW-221 Move and split reporting page #3944 
TW-341 Updates for Environment Merging and PRs #3951

### CODE REVIEW

File: src/pages/docs/getting-started/settings.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ warning: false
 
 Postman automatically chooses default values for some settings so you can get right to work. Make changes to settings at any time based on your use case or to customize your Postman experience.
 
-To change settings in Postman, select the **Settings** button <img alt="Settings icon" src="https://assets.postman.com/postman-docs/settings-icon-v8.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> in the header and then select **Settings**. In the Postman app, you can also select **⌘+Comma (,)** or **Ctrl+Comma (,)**.
+To change settings in Postman, select the **Settings** button <img alt="Settings icon" src="https://assets.postman.com/postman-docs/settings-icon-v8.jpg" width="18px"> in the header and then select **Settings**. In the Postman app, you can also select **⌘+Comma (,)** or **Ctrl+Comma (,)**.
```
Comment (HeatherStenson) [inline src/pages/docs/getting-started/settings.md:40]:
```suggestion
To change settings in Postman, select the **Settings** button <img alt="Settings icon" src="https://assets.postman.com/postman-docs/settings-icon-v8.jpg#icon" width="18px"> in the header and then select **Settings**. In the Postman app, you can also select **⌘+Comma (,)** or **Ctrl+Comma (,)**.
```

File: src/pages/docs/publishing-your-api/viewing-documentation.md [hunk 2]
```diff
@@ -112,11 +112,11 @@ Use the options in the header to customize the appearance of the documentation:
...
 
-> Postman supports [various programming languages and frameworks](/docs/sending-requests/generate-code-snippets/#supported-languagesframeworks) for displaying sample code. Don't see your language of choice, or is there a setting missing that you'd find useful? Select the gear <img alt="Language settings icon" src="https://assets.postman.com/postman-docs/icon-gear-solid-v9.jpg" width="16px" style="vertical-align:middle;margin-bottom:5px"> and then select **Contribute on GitHub** to contribute to [the open-source project](https://github.com/postmanlabs/postman-code-generators).
+> Postman supports [various programming languages and frameworks](/docs/sending-requests/generate-code-snippets/#supported-languagesframeworks) for displaying sample code. Don't see your language of choice, or is there a setting missing that you'd find useful? Select the gear <img alt="Language settings icon" src="https://assets.postman.com/postman-docs/icon-gear-solid-v9.jpg" width="16px"> and then select **Contribute on GitHub** to contribute to [the open-source project](https://github.com/postmanlabs/postman-code-generators).
```
Comment (HeatherStenson) [inline src/pages/docs/publishing-your-api/viewing-documentation.md:119]:
```suggestion
> Postman supports [various programming languages and frameworks](/docs/sending-requests/generate-code-snippets/#supported-languagesframeworks) for displaying sample code. Don't see your language of choice, or is there a setting missing that you'd find useful? Select the gear <img alt="Language settings icon" src="https://assets.postman.com/postman-docs/icon-gear-solid-v9.jpg#icon" width="16px"> and then select **Contribute on GitHub** to contribute to [the open-source project](https://github.com/postmanlabs/postman-code-generators).
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+1. Under **Visibility**, select **Public**.
+1. Select **Update**.
+
+> You must be a workspace [Admin](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to change the visibility of a workspace.
+<!--  -->
+> If you are on a [Postman Professional or Enterprise plan](https://www.postman.com/pricing), you need a approval from a user with the [Community Manager role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to change a workspace's visibility. In the **Overview** tab, select **Workspace Settings**. Under **Visibility**, select **Public** from the dropdown list, then **Request to Change Visibility**.
+
+Once you make a workspace public, all entities within that workspace become publicly accessible through the workspace.
+
+When you make a workspace public, Postman sends in-app and email notifications to all workspace members. To view your in-app notifications, select the notification icon <img alt="Changelog icon" src="https://assets.postman.com/postman-docs/icon-notification-bell-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> in the top right corner of Postman.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:101]:
```suggestion
When you make a workspace public, Postman sends in-app and email notifications to all workspace members. To view your in-app notifications, select the notification icon <img alt="Changelog icon" src="https://assets.postman.com/postman-docs/icon-notification-bell-v9.jpg#icon" width="18px"> in the top right corner of Postman.
```

**Review by HeatherStenson — APPROVED** [review]
Lgtm! 🍊

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+For more information on using a public workspace to help increase adoption of your API, see [Increasing adoption of an API with a public workspace](https://blog.postman.com/increasing-adoption-of-an-api-with-a-public-workspace/) on the Postman blog.
+
+### Sharing a public workspace
+
+You can share a public workspace by giving collaborators its **workspace URL** directly.
+
+To access a workspace's URL:
+
+1. Navigate to the workspace **Overview** page.
+1. Hover over the workspace name to reveal the link icon <img alt="Link icon" src="https://assets.postman.com/postman-docs/icon-workspace-link-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px">.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:126]:
```suggestion
1. Hover over the workspace name to reveal the link icon <img alt="Link icon" src="https://assets.postman.com/postman-docs/icon-workspace-link-v9.jpg#icon" width="18px">.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+The Postman element that you move to the public workspace will no longer exist in the original workspace.
+
+> If there are monitors, mock servers, or integrations associated with the moved element, they remain in the original workspace.
+
+### Deleting elements from a public workspace
+
+You can delete Postman elements from a public workspace for which you have an [Admin or Editor role](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles):
+
+1. Select the three dots icon <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg" width="18px" style="vertical-align:middle;margin-bottom:5px"> next to the element's name, then select **Delete**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/public-workspaces.md:162]:
```suggestion
1. Select the three dots icon <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg#icon" width="18px"> next to the element's name, then select **Delete**.
```

File: src/pages/docs/publishing-your-api/viewing-documentation.md (hunk 2 above)
Comment (mark-anthony-postman) [inline src/pages/docs/publishing-your-api/viewing-documentation.md:119]:
```suggestion
> Postman supports [various programming languages and frameworks](/docs/sending-requests/generate-code-snippets/#supported-languagesframeworks) for displaying sample code. Don't see your language of choice, or is there a setting missing that you'd find useful? Select the gear <img alt="Language settings icon" src="https://assets.postman.com/postman-docs/icon-gear-solid-v9.jpg#icon" width="16px"> and then select **Contribute on GitHub** to contribute to [the open-source project](https://github.com/postmanlabs/postman-code-generators).
```

**Review by mark-anthony-postman — APPROVED** [review]
Some icon issues crept back in--noted in commented. Otherwise LGTM!

**Comment by jkonrath-postman** [comment]
Re all the corrections for the missing `#icon` style - let's do that in another PR.

**Pull request merged**